tests/src/test/scala/ch/renku/acceptancetests/MinimalFunctionalitySpec.scala

```diff
@@ -47,4 +54,15 @@ class MinimalFunctionalitySpec
     launchAnonymousEnvironment map (_ => stopEnvironment(anonEnvConfig.projectId))
 
   }
+
+  def addChangeToProject(implicit projectDetails: ProjectDetails, docsScreenshots: DocsScreenshots): Unit = {
```

Comment:
This looks good, but I think the signature should be:

```
 def addChangeToProject(implicit projectDetails: ProjectDetails): Unit
```

You don't need the docsScreenshots object, since that is only necessary in the hands-on tutorial, where we might want to capture screenshots to update the documentation.